JobScheduler/ApiRequestHandler.cs

```diff
@@ -65,7 +65,7 @@ public ApiRequestHandler(IHttpClientFactory factory, Config config)
         public async Task<List<UserTask>> GetExpectedGraduationUsersAsync()
         {
             HttpResponseMessage response =
-                await client.GetAsync("api/UserTask/CreateUserTasks/" + config.JobSchedulerConfig.TimeRange);
+                await client.GetAsync("localhost/UserTask/CreateUserTasks/" + config.JobSchedulerConfig.TimeRange);
             return JsonConvert.DeserializeObject<List<UserTask>>(await response.Content.ReadAsStringAsync());
         }
 
```

Comment:
Weet niet waar dit voor is maar moet dit ook in de productie env naar localhost staan?